Week-3/Exercise(3.4)/stack.js

```diff
@@ -0,0 +1,17 @@
+function createStack() {
```

Comment:
The solution given is simple yet efficient. Also try implementing the solution using closures.